bundle/manifests/security-profiles-operator.clusterserviceversion.yaml

```
@@ -981,11 +981,11 @@ spec:
   relatedImages:
   - image: gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
     name: rbac-proxy
-  - image: quay.io/security-profiles-operator/selinuxd
+  - image: quay.io/pturrey/v1-security-profiles-operator/selinuxd:latest
```

```suggestion
  - image: quay.io/pturrey/v1-security-profiles-operator/selinuxd:latest
```

switch to p\original